src/__tests__/utils/password.test.ts

```diff
@@ -0,0 +1,25 @@
+import { checkHash, hashFunction } from '../../utils/password';
+import { getRandomString } from '../../__mocks__/getRandom';
+
+describe('testing the hashFunction: ', () => {
+  it('should compare the secret hashed and without', async () => {
+    const secret = getRandomString();
+    expect(await hashFunction(secret)).not.toBe(secret);
+    expect(await hashFunction(secret)).not.toBe(await hashFunction(secret));
+  });
+
+  it('should compare the length of 2 hashed passwords', async () => {;
```

Comment:
these test strings describe what the test is doing but they should describe the desired outcome of the test.
i.e. instead of `'should compare the secret hashed and without'` it should be `'should not return the same hash function twice for the same string'`

i.e. `'should compare the length of 2 hashed passwords'` should be `'should return hashed strings of the same length regardless of the strings passed'`